lib/game.css

```diff
@@ -6,14 +16,35 @@
   display: block;
 }
 
+#game,
+.instructions,
+.difficulty-div {
+  color: white;
+  display: flex;
+}
+
+.instructions,
+.difficulty-div {
+  width: 20%;
+}
+
+.difficulty-div {
+  flex-direction: column;
+}
+
+.easy {
+  margin-bottom: 20px;
+}
+
 h3 {
-  font-size: 50px;
+  color: white;
+  font-size: 30px;
   margin: auto;
   text-align: center;
 }
 
 .score {
-  font-size: 50px;
+  font-size: 30px;
```

Comment:
You're repeating font-sizes here, can you abstract this out to a container element to DRY it up?